### PR TITLE
Date comparisons 2

### PR DESCRIPTION
**Pull request by alexkwolfe**

### CODE REVIEW

**Comment by kellym** [comment]
I see these new operators only apply to the `date` type -- are they going to be supported by `time` as well?

File: src/range.coffee [hunk 1]
```diff
@@ -11,14 +13,25 @@ parse = (string) ->
...
+  if string.match(isoDateRegex)
+    parts = []
+
+    while (m = isoDateRegex.exec(string)) != null
+      parts.push m[0]
+
+    getTime = (s) ->
+      new Date(s).getTime().toString()
+
+    string = parts.map(getTime).join('-')
```
Comment (alexkwolfe) [inline src/range.coffee]: This is a stupid edge case, but what if there are three times?

File: src/date.coffee [hunk 2]
```diff
@@ -31,8 +31,14 @@ module.exports =
...
+    'is less than'
+    'is less than or equal to'
+    'is greater than'
+    'is greater than or equal to'
     'is blank'
     'is not blank'
     'format is valid'
     'format is invalid'
+    'is between'
+    'is not between'
```
Comment (alexkwolfe) [inline src/date.coffee]: I think the time type also needs these operators.

**Comment by alexkwolfe** [comment]
LGTM

File: src/range.coffee (hunk 1 above)
Comment (bhicks) [inline src/range.coffee]: We should guard against that. I'll change it to only take the first two available.

**Pull request merged**